app/scripts/modules/netflix/tableau/tableau.dataSource.ts

```diff
@@ -1,14 +1,15 @@
+import {module} from 'angular';
+
 import {DataSourceConfig} from 'core/application/service/applicationDataSource';
-import {APPLICATION_DATA_SOURCE_REGISTRY} from 'core/application/service/applicationDataSource.registry';
+import {
+  APPLICATION_DATA_SOURCE_REGISTRY,
+  ApplicationDataSourceRegistry
+} from 'core/application/service/applicationDataSource.registry';
```

Comment:
My 2 cents: on collaborative projects, I prefer choosing a "rule" that the linter can enforce. 

I don't much care what that rule is, but hopefully it's one that IntelliJ can apply using `cmd-option-O` (optimize imports)

IntelliJ offers 2 choices: "wrap if long" and "chop down if long" (I personally prefer "wrap if long")